### DISCUSSION TITLE
Proper motion values as displayed vs that in the catalog.

### DISCUSSION THREAD

**Discussion by BikramAjay** [Stellarium/stellarium#1778] [General]
Why the proper motion values as displayed in the catalog and by Stellarium are different? 

H11 :       41.73          Trigonometric parallax (mas)
H12 :      121.23          mu_alpha* (mas/yr)
H13 :      -22.01          mu_delta (mas/yr)

https://hipparcos-tools.cosmos.esa.int/cgi-bin/HIPcatalogueSearch.pl?hipId=65378%20A&hdId=&tyc1=&tyc2=&tyc3=

![Screenshot 2021-07-01 045243](https://user-images.githubusercontent.com/86772525/124097885-03132500-da2a-11eb-8c4c-34c2c514976f.jpg)

**Comment by alex-w** [comment]
Because this is error in our catalog

**Reply by BikramAjay** [reply]
How come every star in the Big Dipper constellation has wrong PM values? I haven't checked for other stars.

**Reply by gzotti** [reply]
catalog versions?

**Reply by BikramAjay** [reply]
I don't know. 
The search tool is at the link below.
https://www.cosmos.esa.int/web/hipparcos/search-facility

When clicked it gives this search interface.
https://hipparcos-tools.cosmos.esa.int/HIPcatalogueSearch.html

when entered HIP number 65378 for Mizar.

I get this. I don't see any version number here.  Even if we assume that PM values are the current ie to date then will it change that much in just 30 years (1991-2021)?
https://hipparcos-tools.cosmos.esa.int/cgi-bin/HIPcatalogueSearch.pl?hipId=65378&hdId=&tyc1=&tyc2=&tyc3=

![Screenshot 2021-07-01 135706](https://user-images.githubusercontent.com/86772525/124169536-56aa6080-da74-11eb-8d40-e73638ef1f0d.jpg)
![Screenshot 2021-07-01 135732](https://user-images.githubusercontent.com/86772525/124169544-590cba80-da74-11eb-87d5-6c3303e7bca5.jpg)

**Reply by BikramAjay** [reply]
Any answers?

**Reply by alex-w** [reply]
I filled bug report: https://github.com/Stellarium/stellarium/issues/1787

**Reply by alex-w** [reply]
Quote from description:
> Fields H12–13: Proper motion components (epoch J1991.25, ICRS)
> The proper motion components, µα* = µα cos δ and µδ , are expressed in milliarcsec per Julian year (mas/yr), and are given with respect to the reference system ICRS.

**Reply by BikramAjay** [reply]
Thanks

**Comment by alex-w** [comment]
Proper motion in our star catalog are encoded and, of course, we shouldn't just display it, because these values will have wrong data. :(

I'll fix the displayed values of PM.
